src/go/rdctl/cmd/snapshotClean.go

```diff
@@ -0,0 +1,23 @@
+package cmd
```

Comment:
Why is this command needed? Is it a temporary placeholder until we know there are no bugs, or is there a reason why I would want to clean a snapshot? And I would call this command `remove-lock` just to make the name reflect the action, unless we expect more cleaning operations will be needed in the future.

In short, a docstring is needed above `var snapshotCleanCmd` to explain what this is for

Reply:
If you look at the "How it Works Internally" that Jan wrote up [in the epic](https://github.com/rancher-sandbox/rancher-desktop/issues/4236), he mentions a `rdctl snapshot unlock` command. That's what this is supposed to be - I called it `clean` because that's what I had in my brain, but I will update it to `unlock`.

I will also add a docstring, and a `Long` field to the command as requested by @mook-as.

Reply:
Actually, the `Long` field should be sufficient documentation - I'm going to hold off on the docstring if that's ok.